ci/pipeline.yml

```diff
@@ -11,17 +11,26 @@ email-cf: &email-cf
   CF_STACK: cflinuxfs4
 
 node-image: &node-image
```

Comment:
This happened to me while updating the redirects CI task. 

You'll want to configure it to look like:
```yml
  config:
    <<: *node-image
```

and not 
```yml
  config:
    platform: linux
    image_resource:
      <<: *node-image
```

We added the `platform` and `image_resource` keys to the `node-image` anchor so we don't need to call them again when using it in a task.